### PR TITLE
Code cleanup--tests

### LINKED ISSUE #28964: Reformat code
Before closing 7.0

### PR DESCRIPTION
**Pull request by ajcvickers**
Fixes #28964

### CODE REVIEW

File: test/EFCore.Relational.Specification.Tests/Update/StoredProcedureUpdateTestBase.cs [hunk 1]
```diff
@@ -481,10 +480,7 @@ public virtual async Task Original_and_current_value_on_non_concurrency_token(bo
     {
         await using var context = CreateContext();
 
-        var entity = new Entity
-        {
-            Name = "Initial",
-        };
+        var entity = new Entity { Name = "Initial", };
```
Comment (roji) [inline test/EFCore.Relational.Specification.Tests/Update/StoredProcedureUpdateTestBase.cs:483]:
```suggestion
        var entity = new Entity { Name = "Initial" };
```

File: test/EFCore.Relational.Tests/Update/ModificationCommandComparerTest.cs [hunk 2]
```diff
@@ -34,7 +34,7 @@ public void Compare_returns_0_only_for_commands_that_are_equal()
         entry1.SetEntityState(EntityState.Added);
         var modificationCommandAdded = modificationCommandSource.CreateModificationCommand(
             new ModificationCommandParameters(table, false, false, null, new ParameterNameGenerator().GenerateNext));
-            // new ModificationCommandParameters("A", null, false, false, null, new ParameterNameGenerator().GenerateNext));
+        // new ModificationCommandParameters("A", null, false, false, null, new ParameterNameGenerator().GenerateNext));
```
Comment (roji) [inline test/EFCore.Relational.Tests/Update/ModificationCommandComparerTest.cs:37]:
```suggestion
```

**Review by roji — APPROVED** [review]

File: test/EFCore.Relational.Tests/Update/ModificationCommandComparerTest.cs (hunk 2 above)
Comment (ajcvickers) [reply test/EFCore.Relational.Tests/Update/ModificationCommandComparerTest.cs:37]: fixed

File: test/EFCore.Relational.Specification.Tests/Update/StoredProcedureUpdateTestBase.cs (hunk 1 above)
Comment (ajcvickers) [reply test/EFCore.Relational.Specification.Tests/Update/StoredProcedureUpdateTestBase.cs:483]: Fixed

**Pull request merged**